.npmrc

```diff
@@ -0,0 +1,3 @@
+registry=http://registry.npmjs.org/
+@foxden:registry=https://registry.npmjs.org/
+//registry.npmjs.org/:_authToken=${NPM_TOKEN}
```

Comment:
You'll probably also want an .npmignore file to make sure we're not bundling files that we shouldn't be.